pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -187,23 +198,39 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, "", 0, describeServiceInServiceGroup(f, service, exposes...)...)
 
 			for _, dcPipeline := range service.DeploymentConfigPipelines {
-				printLines(out, indent, 1, describeDeploymentInServiceGroup(local, dcPipeline, func(rc *kubegraph.ReplicationControllerNode) int32 {
+				printLines(out, indent, 1, describeConfigDeploymentsInServiceGroup(local, dcPipeline, func(rc *kubegraph.ReplicationControllerNode) int32 {
```

Comment:
@kargakis in future, we would like to keep just KubeDeployments right? it will cause another headache then ;-)